app/Console/Commands/SendConfirmations.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\MyMuseumTour;
+use Illuminate\Console\Command;
+
+class SendConfirmations extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'send:confirmations';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command to go through My Museum Tours and send email confirmations';
+
+    /**
+     * Array of models to generate PDFs for. Format is `route => model class`
+     */
+    protected static array $models = [
+        'my-museum-tour.show' => MyMuseumTour::class,
+    ];
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        foreach (self::$models as $route => $modelClass) {
+            $models = $modelClass::notSent()->get();
+
+            foreach ($models as $model) {
+                try {
+                    $this->sendConfirmation($model, $route);
+                } catch (\Exception $exception) {
+                    $this->error($exception->getMessage());
+                    return 1;
```

Comment:
This will stop at the first error. Is that the desired behavior?

Reply:
Good question. Without the `try` block in here I think it would have the same behavior, so I think this is redundant. I'll remove it.